src/commands/messageContent/AutoClearChannels.ts

```diff
@@ -0,0 +1,61 @@
+import { Client, Guild, Message, TextChannel } from "discord.js";
+import { configuration } from "../../server";
+
+export default {
+    trigger: /./,
+    handler: (message: Message) => {
+        if (shouldClearChannel(message)) {
+            deleteMessages(message);
+        }
+    },
+};
+
+export async function clearChannels(client: Client) {
+    async function clearChannel(guild: Guild, channelId: string) {
+        const channelToClear = guild.channels.cache.find(({ id }) => id == channelId) as TextChannel;
+        if (!channelToClear) {
+            return;
+        }
+
+        const last = (await channelToClear?.messages.fetch({ limit: 1 })).first();
+        if (!last) {
+            return;
+        }
+
+        const date = new Date();
+        const hours_since = (date.getTime() - last.createdAt.getTime()) / 1000 / 60 / 60;
```

Comment:
(1000 * 60 * 60) should be extracted to a constant, or this entire line should just be a helper method.

Reply:
I've actually moved this setting entirely to the JSON settings file.